.github/workflows/nodejs.yml

```diff
@@ -4,7 +4,7 @@
 name: Node.js CI
 
 env:
-  RIPPLED_DOCKER_IMAGE: rippleci/rippled:2.3.0-rc1
+  RIPPLED_DOCKER_IMAGE: rippleci/rippled:develop
```

Comment:
Hm I like this change - less updating needed in these files, and it serves as an additional form of testing for rippled

Reply:
I would say we point to a specific version because when `develop` updates, our integ tests may fail. Also, it's helpful to see what version of rippled the client libraries are testing against.